<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="sendbutton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>631</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="msgEditor">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>{007,232,001,200,8,10,}</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="packetVisualizer">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>448</y>
      <width>321</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="msgStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="allPacketVisualizer">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>450</y>
      <width>321</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>410</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QLabel" name="deviceIdLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Device id</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deviceIdBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltage</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="VoltageBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>15</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CurrentBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Current</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CurrentViewerBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="currentViewerLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Current</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="VoltageViewerBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="VoltageViewerLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="turnOffButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TurnOff</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="measurementQueryRadio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Measurement Query</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="measurementCapacityRadio">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Measurement Capacity</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="pingRadio">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ping</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="setParametersRadio">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>set Parameters</string>
    </property>
   </widget>
   <widget class="QLabel" name="msgStatus_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="temprature2ViewerLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>270</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Temprature2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="temprature2ViewerBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>270</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Temprature1ViewerBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>270</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Temprature1ViewerLabel">
    <property name="geometry">
     <rect>
      <x>326</x>
      <y>270</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Temprature1</string>
    </property>
   </widget>
   <widget class="QLabel" name="msgStatus_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="exceptionViewerLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>330</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Exception</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="exceptionViewerBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>330</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="capacityViewerBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>330</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="capacityViewerLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Capacity</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rawFrameRadio">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>RawFrame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="turnOnButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TurnOn</string>
    </property>
   </widget>
   <widget class="QLabel" name="deviceIdViewerLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>330</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Device id</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deviceIdViewerBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>330</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sentPacket">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>400</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="RepeatRequest">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Repeat Request</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>686</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
